specification/compute/resource-manager/readme.md

```diff
@@ -37,6 +37,14 @@ openapi-type: arm
 tag: package-2017-03
 ```
 
+### Tag: package-2017-10-15-preview
+
+These settings apply only when `--tag=package-2017-10-15-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2017-10-15-preview'
+input-file:
+- Microsoft.Compute/2017-10-15-preview/computeArtifactPublishingService.json

```

Comment:
what you list here is what a "Compute SDK" would now contain - you probably wanna add
```
 - Microsoft.Compute/2017-03-30/compute.json
 - Microsoft.Compute/2017-03-30/disk.json
 - Microsoft.Compute/2017-03-30/runCommands.json
 - Microsoft.ContainerService/2017-01-31/containerService.json
```
to the list.